Tox checks run just fine with Python 3.9, but the tox configuration doesn't
define the environment. Define it here until upstream does.

--- tox.ini.bak	2020-10-16 12:21:22.109040944 -0400
+++ tox.ini	2020-10-16 12:21:56.390042112 -0400
@@ -5,7 +5,7 @@
 
 [tox]
 minversion = 2.0
-envlist = py27,py3{4,5,6,7,8},py{27,38}-flake8,pypy
+envlist = py27,py3{4,5,6,7,8,9},py{27,38,39}-flake8,pypy
 
 [testenv]
 basepython =
@@ -15,6 +15,7 @@
     py36: {env:TOXPYTHON:python3.6}
     py37: {env:TOXPYTHON:python3.7}
     py38: {env:TOXPYTHON:python3.8}
+    py39: {env:TOXPYTHON:python3.9}
     pypy: {env:TOXPYTHON:pypy}
 commands =
     {envpython} --version
@@ -63,6 +64,10 @@
 commands =
     {[tests]commands}
 
+[testenv:py39]
+commands =
+    {[tests]commands}
+
 [testenv:py27-flake8]
 deps =
     flake8
@@ -74,6 +79,13 @@
 deps =
     flake8
 commands =
+    {[testenv]commands}
+    flake8 .
+
+[testenv:py39-flake8]
+deps =
+    flake8
+commands =
     {[testenv]commands}
     flake8 .
 
